### DISCUSSION TITLE
Versionning

### DISCUSSION THREAD

**Discussion by ArchieCoder** [unoplatform/uno#18469] [General]
Hello, our app is built with a pipeline. I was not sure how to do the versionning and I suggested that we update:
- csproj: ApplicationDisplayVersion and ApplicationVersion to 1.0.0.0
- Package.appxmanifest: Identity/Version to something 1.0.0.0

There is a app.manifest file that has assemblyIdentity with a version 1.0.0.0.

With the new Uno.Sdk, there is a new validation that requires ApplicationVersion to be x.y and ApplicationVersion to x.

There is nothing in the documentation about the previous two IDs.

For simplicity, would it be possible if Uno only check Package.appxmanifest version only?

**Comment by morning4coffe-dev** [comment]
Hey @ArchieCoder,

Thank you for asking this question, this would be very nice to have, but not completely sure, if possible. I recently had some issues with Android versioning https://github.com/unoplatform/uno/issues/18099. I think, it would be nice, if the csproj version would propagate to the appxmanifest as well, @MartinZikmund what do you think?
